src/m365/spo/commands/field/field-get.spec.ts

```diff
@@ -320,6 +320,51 @@ describe(commands.FIELD_GET, () => {
     assert.strictEqual(getStub.lastCall.args[0].url, 'https://contoso.sharepoint.com/sites/portal/_api/web/lists(guid\'03e45e84-1992-4d42-9116-26f756012634\')/fields/getbyinternalnameortitle(\'Title\')');
   });
 
+  it('should call the correct GET url when field internalName and list title specified (verbose)', async () => {
+    const getStub = sinon.stub(request, 'get').callsFake(async (opts) => {
+      if ((opts.url as string).indexOf(`/_api/web/lists`) > -1) {
+        return {
+          "Id": "03e45e84-1992-4d42-9116-26f756012634"
+        };
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, { options: { debug: true, verbose: true, webUrl: 'https://contoso.sharepoint.com/sites/portal', internalName: 'Title', listTitle: 'Documents' } });
+    assert.strictEqual(getStub.lastCall.args[0].url, 'https://contoso.sharepoint.com/sites/portal/_api/web/lists/getByTitle(\'Documents\')/fields/getbyinternalnameortitle(\'Title\')');
+  });
+
+  it('should call the correct GET url when field internalName and list title specified', async () => {
+    const getStub = sinon.stub(request, 'get').callsFake(async (opts) => {
+      if ((opts.url as string).indexOf(`/_api/web/lists`) > -1) {
+        return {
+          "Id": "03e45e84-1992-4d42-9116-26f756012634"
+        };
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, { options: { webUrl: 'https://contoso.sharepoint.com/sites/portal', internalName: 'Title', listTitle: 'Documents' } });
+    assert.strictEqual(getStub.lastCall.args[0].url, 'https://contoso.sharepoint.com/sites/portal/_api/web/lists/getByTitle(\'Documents\')/fields/getbyinternalnameortitle(\'Title\')');
+  });
+
+  it('should call the correct GET url when field internalName and list url specified', async () => {
+    const getStub = sinon.stub(request, 'get').callsFake(async (opts) => {
+      if ((opts.url as string).indexOf(`/_api/web/lists`) > -1) {
```

Comment:
Could we please use full URL's instead of using indexOf please? Please do this for all your tests